js/content/store.js

```diff
@@ -1385,10 +1385,24 @@ class AppPageClass extends StorePageClass {
         if (this.isDlc()) { return; }
 
         let appid = this.appid;
-        let response = await Background.action("appdetails", {"appids": appid, "filters": "support_info"});
-        if (!response || !response[appid] || !response[appid].success) { return; }
+        let cache = LocalStorage.get("support_infos", {});
+        for (let id in cache) {
+            if (!cache[id].expiry || cache[id].expiry < Date.now()) {
+                delete cache[id];
+            }
+        }
+
+        let supportInfo = cache[appid];
+        if (!supportInfo) {
+            let response = await Background.action("appdetails", {"appids": appid, "filters": "support_info"});
+            if (!response || !response[appid] || !response[appid].success) { return; }
+
+            supportInfo = response[appid].data.support_info;
+            supportInfo.expiry = Date.now() + 2592000000; // 30 days
+            cache[appid] = supportInfo;
+            LocalStorage.set("support_infos", cache);
```

Comment:
Why cache here instead of the Background?

Reply:
For now, this is ok.